lib/chef/resource/windows_package.rb

```diff
@@ -26,13 +26,12 @@ class Resource
     class WindowsPackage < Chef::Resource::Package
       include Chef::Mixin::Uris
 
-      provides :package, os: "windows"
       provides :windows_package, os: "windows"
+      provides :package, os: "windows"
```

Comment:
(I think the answer to that particular question is "we could, but it's unclear which is better and I'm trying not to fix things I don't have a clear answer for right now :)")